packages/app-toolbox/src/Unlock.tsx

```diff
@@ -75,72 +52,61 @@ class Unlock extends React.PureComponent<Props, State> {
         <Modal.Header>
           {t('Unlock account')}
         </Modal.Header>
-        <Modal.Content>
-          {this.renderContent()}
-        </Modal.Content>
-        <Modal.Actions>
-          {this.renderActions()}
-        </Modal.Actions>
+        {this.renderContent()}
+        {this.renderActions()}
       </Modal>
     );
   }
 
-  renderActions () {
+  private renderActions () {
     const { t } = this.props;
 
     return (
-      <Button.Group>
-        <Button
-          isNegative
-          onClick={this.onCancel}
-          label={t('Cancel')}
-        />
-        <Button.Or />
-        <Button
-          isPrimary
-          onClick={this.onUnlock}
-          label={t('Unlock')}
-        />
-      </Button.Group>
+      <Modal.Actions>
+        <Button.Group>
+          <Button
+            isNegative
+            onClick={this.onCancel}
+            label={t('Cancel')}
+          />
+          <Button.Or />
+          <Button
+            isPrimary
+            onClick={this.onUnlock}
+            label={t('Unlock')}
+          />
+        </Button.Group>
+      </Modal.Actions>
     );
   }
 
-  renderContent () {
+  private renderContent () {
     const { t } = this.props;
     const { address, password, unlockError } = this.state;
 
     return (
-      <>
-        <WrapperContent>
-          <div className='expanded'>
-            <p>
-              <Trans>
-                You are about to unlock your account <span className='code'>{address}</span> to allow for the signing of messages.
-              </Trans>
-            </p>
-          </div>
-          <IdentityIcon
-            className='icon'
-            value={address}
-          />
-        </WrapperContent>
-        <WrapperEntry>
-          <div className='ui--row'>
+      <Modal.Content>
+        <AddressRow
+          isInline
+          value={address}
+        >
+          <p>{t('You are about to unlock your account to allow for the signing of messages. Once active the signature will be generated based on the content provided.')}</p>
+          <div>
             <Password
-              className='medium'
+              autoFocus
               isError={!!unlockError}
               help={t('The account\'s password specified at the creation of this account.')}
               label={t('unlock account using')}
```

Comment:
With the additional text, I'd say it's clear that the account will be unlocked.

```suggestion
              label={t('password')}
```